root/src/main/java/com/example/servingwebcontent/Movie.java

```diff
@@ -32,8 +36,16 @@ public String getFilmRating () {
 		return filmRating;
 	}
 
-	public Movie (String id, String type, String name, String releaseDate, int duration, String filmRating) {
-		this.id = id;
+	public Movie (String type, String name, String releaseDate, int duration, String filmRating) {
+		this.type = type;
+		this.name = name;
+		this.releaseDate = releaseDate;
+		this.duration = duration;
+		this.filmRating = filmRating;
+	}
+
+	public Movie (String movieID, String type, String name, String releaseDate, int duration, String filmRating) {
+		this.movieID = movieID;
```

Comment:
you should use "this" to reuse your other constructor. In other words, something like:

this(type, name, releaseDate, duration, filmRating);
this.movieId = movieId;